recipes/shapeshifter/meta.yaml

```diff
@@ -1,28 +1,42 @@
 {% set name = "shapeshifter" %}
-{% set version = "0.0.3" %}
+{% set version = "1.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9679af30022df7eeeafa11d9f8dd62408417bb439efb343f272de615eadacd33" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: dea83a20033b75e35cd4483b8703d322f81315a448133615f9254d75a03c1924
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
 
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   host:
-    - pip
     - python >=3
-  run:
+    - setuptools
     - pandas
     - pyarrow
+    - numpy ==1.15.4
```

Comment:
I think that should be `>=1.15.4` so that it will allow newer versions as well. If the package does really special stuff, `>=1.15.4,<1.16a0`  would make it allow only that range.

Reply:
Also, if the package can work with older numpy, please remove this. The current default pinngs are for numpy 1.9 - that's what all binary packages are built against. See link below.

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/9b8e67fcac4a4199dc7af7f120564aceab1cc7b8/recipe/conda_build_config.yaml#L516-L517

Reply:
Previously, numpy was not working correctly with hdf files in versions other than 1.15, but has been patched to fix this.

Version has now been changed to any